app/assets/javascripts/components/nav_menu.js.jsx

```diff
@@ -0,0 +1,34 @@
+var React = require('react');
+
+var NavMenu = React.createClass({
+
+  getInitialState: function(){
+    return { focused: 0 };
+  },
+
+  clicked: function(index){
+    this.setState({focused: index});
+  },
+
+  render: function() {
+    var self = this;
+    return (
+      <span>
+      <ul>{
```

Comment:
Let's take all this logic out of the JSX block.

Also, try to use consistent whitespace. 2 spaces for indents. The [Airbnb style guide](http://nerds.airbnb.com/our-javascript-style-guide/) is a good resource.